src/adagrams.js

```diff
@@ -1,8 +1,125 @@
+// 1. Set up array data structure:
+const pool = Array(9).fill('A').concat(
```

Comment:
This data structure works, but it's not very DRY and would be tricky to change. For example, suppose I said you had the wrong number of "I"s - you would have to do a lot of counting to solve the problem.

Instead you might store a hash of letter frequencies like this

```
letterQuantities: {
  A: 9, B: 2, ...
}
```

and use it to build a pool of letters dynamically.